lib/core/public/cleanup-plugins.js

```diff
@@ -23,9 +23,11 @@ function cleanupPlugins(resolve, reject) {
     });
   });
 
-  axe.utils.toArray(document.querySelectorAll('frame, iframe')).forEach(function (frame) {
+  var flattenedTree = axe.utils.getFlattenedTree(document.body)
+
+  axe.utils.querySelectorAll(flattenedTree, 'iframe, frame').forEach(function (node) {
     q.defer(function (res, rej) {
-      return axe.utils.sendCommandToFrame(frame, {
+      return axe.utils.sendCommandToFrame(node.actualNode, {

```

Comment:
This is super tweaky, but I wonder if it would be more intuitive to write `virtualNode.actualNode` or `vNode.actualNode` so it communicates what that top level object is. That would help in other places too (but changing it everywhere is obvs outside the scope of this PR)